### PR TITLE
chore(curriculum): update book inventory app tests

### PR DESCRIPTION
**Pull request by a2937**
Checklist:

- [X] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [X] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

This updates the assertions in the lab book inventory app to use proper assertion methods.

### CODE REVIEW

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 1]
```diff
@@ -247,13 +245,13 @@ assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"] span
 You should use an attribute selector to target the `span` elements with the class of `status` that are descendants of `tr` elements with the class of `to-read` and set their `border` property.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"] span[class="status"]')?.border);
+assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"] span[class="status"]')?.border);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:248]:
```suggestion
assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"] span[class="status"]')?.border);
```

properties are all present, if they do not have a value it's an empty string

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 2]
```diff
@@ -265,13 +263,13 @@ assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="read"] span[cl
 You should use an attribute selector to target the `span` elements with the class of `status` that are descendants of `tr` elements with the class of `read` and set their `border` property.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('tr[class="read"] span[class="status"]')?.border);
+assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="read"] span[class="status"]')?.border);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:266]:
```suggestion
assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('tr[class="read"] span[class="status"]')?.border);
```

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 3]
```diff
@@ -265,13 +263,13 @@ assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="read"] span[cl
...
 ```
 
 You should use an attribute selector to target the `span` elements with the class of `status` that are descendants of `tr` elements with the class of `read` and set their `background-image` property.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('tr[class="read"] span[class="status"]')?.backgroundImage);
+assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="read"] span[class="status"]')?.backgroundImage);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:272]:
```suggestion
assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('tr[class="read"] span[class="status"]')?.backgroundImage);
```

and so on please every time there is a property being checked

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
`exists` is not appropriate for testing a property from `getStyle`, all the properties are always strings, empty if there is no value. Please use `assert.isNotEmpty()`

**Comment by a2937** [comment]
I understand.

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 4]
```diff
@@ -283,13 +281,13 @@ assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="in-progress"]
...
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('tr[class="in-progress"] span[class="status"]')?.border);
+assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="in-progress"] span[class="status"]')?.border);
 ```
 
 You should use an attribute selector to target the `span` elements with the class of `status` that are descendants of `tr` elements with the class of `in-progress` and set their `background-image` property.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('tr[class="in-progress"] span[class="status"]')?.backgroundImage);
+assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="in-progress"] span[class="status"]')?.backgroundImage);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:290]: here

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 5]
```diff
@@ -305,23 +303,23 @@ You should use an attribute selector to target the `span` elements with the clas
 ```js
 const selector1 = new __helpers.CSSHelp(document).getStyle('span[class="status"], span[class^="rate"]');
 const selector2 = new __helpers.CSSHelp(document).getStyle('span[class^="rate"], span[class="status"]');
-assert(selector1?.height || selector2?.height);
+assert.exists(selector1?.height || selector2?.height);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:306]: here

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 6]
```diff
@@ -305,23 +303,23 @@ You should use an attribute selector to target the `span` elements with the clas
...
 ```
 
 You should use an attribute selector to target the `span` elements with the class of `status` and the `span` elements with the class value starting with `rate` and set their `width` property.
 
 ```js
 const selector1 = new __helpers.CSSHelp(document).getStyle('span[class="status"], span[class^="rate"]');
 const selector2 = new __helpers.CSSHelp(document).getStyle('span[class^="rate"], span[class="status"]');
-assert(selector1?.width || selector2?.width);
+assert.exists(selector1?.width || selector2?.width);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:314]: here

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 7]
```diff
@@ -305,23 +303,23 @@ You should use an attribute selector to target the `span` elements with the clas
...
 ```
 
 You should use an attribute selector to target the `span` elements with the class of `status` and the `span` elements with the class value starting with `rate` and set their `padding` property.
 
 ```js
 const selector1 = new __helpers.CSSHelp(document).getStyle('span[class="status"], span[class^="rate"]');
 const selector2 = new __helpers.CSSHelp(document).getStyle('span[class^="rate"], span[class="status"]');
-assert(selector1?.padding || selector2?.padding);
+assert.exists(selector1?.padding || selector2?.padding);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:322]: here

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 8]
```diff
@@ -333,37 +331,37 @@ assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > sp
 You should use an attribute selector to target the `span` elements which are direct children of `span` elements with the `class` value starting with `rate` and set their `border` property.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span')?.getPropVal('border'));
+assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span')?.getPropVal('border'));
 ```
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:335]: here

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 9]
```diff
@@ -333,37 +331,37 @@ assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > sp
...
 
 You should use an attribute selector to target the `span` elements which are direct children of `span` elements with the `class` value starting with `rate` and set their `border-radius` property.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span')?.getPropVal('border-radius'));
+assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span')?.getPropVal('border-radius'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:340]: here

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 10]
```diff
@@ -333,37 +331,37 @@ assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > sp
...
 ```
 
 You should use an attribute selector to target the `span` elements which are direct children of `span` elements with the `class` value starting with `rate` and set their `margin` property.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span')?.getPropVal('margin'));
+assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span')?.getPropVal('margin'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:346]: here

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 11]
```diff
@@ -333,37 +331,37 @@ assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > sp
...
 ```
 
 You should use an attribute selector to target the `span` elements which are direct children of `span` elements with the `class` value starting with `rate` and set their `height` property.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span')?.getPropVal('height'));
+assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span')?.getPropVal('height'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:352]: here

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 12]
```diff
@@ -333,37 +331,37 @@ assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > sp
...
 ```
 
 You should use an attribute selector to target the `span` elements which are direct children of `span` elements with the `class` value starting with `rate` and set their `width` property.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span')?.getPropVal('width'));
+assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span')?.getPropVal('width'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:358]: here

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 13]
```diff
@@ -333,37 +331,37 @@ assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > sp
...
 ```
 
 You should use an attribute selector to target the `span` elements which are direct children of `span` elements with the `class` value starting with `rate` and set their `background-color` property.
 
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span')?.getPropVal('background-color'));
+assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span')?.getPropVal('background-color'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:364]: here

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 14]
```diff
@@ -241,145 +238,145 @@ assert.equal(new __helpers.CSSHelp(document).getStyle('span')?.getPropVal('displ
...
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('tr[class="in-progress"] span[class="status"]')?.backgroundImage);
 ```
 
 You should have an attribute selector to target the `span` elements with the class of `status` and the `span` elements with the class value starting with `rate`.
 
 ```js
 const selector1 = new __helpers.CSSHelp(document).getStyle('span[class="status"], span[class^="rate"]');
 const selector2 = new __helpers.CSSHelp(document).getStyle('span[class^="rate"], span[class="status"]');
-assert.exists(selector1 || selector2);
+assert.isNotEmpty(selector1 || selector2);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:297]:
```suggestion
assert.exists(selector1 || selector2);
```

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 15]
```diff
@@ -241,145 +238,145 @@ assert.equal(new __helpers.CSSHelp(document).getStyle('span')?.getPropVal('displ
...
+assert.isNotEmpty(selector1?.height || selector2?.height);
 ```
 
 You should use an attribute selector to target the `span` elements with the class of `status` and the `span` elements with the class value starting with `rate` and set their `width` property.
 
 ```js
 const selector1 = new __helpers.CSSHelp(document).getStyle('span[class="status"], span[class^="rate"]');
 const selector2 = new __helpers.CSSHelp(document).getStyle('span[class^="rate"], span[class="status"]');
-assert(selector1?.width || selector2?.width);
+assert.isNotEmpty(selector1?.width || selector2?.width);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:313]:
```suggestion
assert.exists(selector1?.width || selector2?.width);
```

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 16]
```diff
@@ -241,145 +238,145 @@ assert.equal(new __helpers.CSSHelp(document).getStyle('span')?.getPropVal('displ
...
 const selector2 = new __helpers.CSSHelp(document).getStyle('span[class^="rate"], span[class="status"]');
-assert(selector1?.padding || selector2?.padding);
+assert.isNotEmpty(selector1?.padding || selector2?.padding);
 ```
 
 You should have an attribute selector to target the `span` elements which are direct children of `span` elements with the `class` value starting with `rate`.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span'));
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:327]:
```suggestion
assert.exists(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span'));
```

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 17]
```diff
@@ -241,145 +238,145 @@ assert.equal(new __helpers.CSSHelp(document).getStyle('span')?.getPropVal('displ
...
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('span[class^="rate"] > span')?.getPropVal('background-color'));
 ```
 
 You should have an attribute selector to target the first descendant of `span` elements that have `one` as a part of their `class` value.
 
 ```js
 const selector1 = new __helpers.CSSHelp(document).getStyle('span[class~="one"] :first-child');
 const selector2 = new __helpers.CSSHelp(document).getStyle('span[class~="one"] :nth-child(1)');
-assert.exists(selector1 || selector2);
+assert.isNotEmpty(selector1 || selector2);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:371]:
```suggestion
assert.exists(selector1 || selector2);
```

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 18]
```diff
@@ -389,7 +386,7 @@ const selector1 = new __helpers.CSSHelp(document).getStyle('span[class~="two"] :
 const selector2 = new __helpers.CSSHelp(document).getStyle('span[class~="two"] :nth-child(2), span[class~="two"] :nth-child(1)');
 const selector3 = new __helpers.CSSHelp(document).getStyle('span[class~="two"] :first-child, span[class~="two"] :nth-child(2)');
 const selector4 = new __helpers.CSSHelp(document).getStyle('span[class~="two"] :nth-child(2), span[class~="two"] :first-child');
-assert.exists(selector1 || selector2 || selector3 || selector4);
+assert.isNotEmpty(selector1 || selector2 || selector3 || selector4);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:389]:
```suggestion
assert.exists(selector1 || selector2 || selector3 || selector4);
```

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 19]
```diff
@@ -400,19 +397,19 @@ const selector2 = new __helpers.CSSHelp(document).getStyle('span[class~="two"] :
...
 }
-assert(isTrue);
+assert.isTrue(containsSelector); 
 ```
 
 You should have an attribute selector to target the `span` elements that are descendants of `span` elements that have `three` as a part of their `class` value.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('span[class~="three"] span'));
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('span[class~="three"] span'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:412]:
```suggestion
assert.exists(new __helpers.CSSHelp(document).getStyle('span[class~="three"] span'));
```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 20]
```diff
@@ -192,14 +189,14 @@ for (let e of rateSpans) {
...
-    assert(['one', 'two', 'three'].includes(e.classList[1]));
+    assert.oneOf(e.classList[1], ['one', 'two', 'three']);
 }
 ```
 
 You should have an attribute selector to target rows that have the class of `read`.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="read"]'));
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('tr[class="read"]'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:199]: `getStyle` returns `null` if the selector is not there. `exists` or `isNotNull` are more appropriate

```suggestion
assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="read"]'));
```

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 21]
```diff
@@ -211,7 +208,7 @@ assert.include(new __helpers.CSSHelp(document).getStyle('tr[class="read"]')?.bac
 You should have an attribute selector to target rows that have the class of `to-read`.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"]'));
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"]'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:211]:
```suggestion
assert.exist(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"]'));
```

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 22]
```diff
@@ -223,7 +220,7 @@ assert.include(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"]')?.
 You should have an attribute selector to target rows that have the class of `in-progress`.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="in-progress"]'));
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('tr[class="in-progress"]'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:223]:
```suggestion
assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="in-progress"]'));
```

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 23]
```diff
@@ -241,55 +238,55 @@ assert.equal(new __helpers.CSSHelp(document).getStyle('span')?.getPropVal('displ
 You should have an attribute selector to target the `span` elements with the class of `status` that are descendants of `tr` elements with the class of `to-read`.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"] span[class="status"]'));
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"] span[class="status"]'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:241]:
```suggestion
assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"] span[class="status"]'));
```

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 24]
```diff
@@ -241,55 +238,55 @@ assert.equal(new __helpers.CSSHelp(document).getStyle('span')?.getPropVal('displ
...
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"] span[class="status"]')?.backgroundImage);
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"] span[class="status"]')?.backgroundImage);
 ```
 
 You should have an attribute selector to target the `span` elements with the class of `status` that are descendants of `tr` elements with the class of `read`.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="read"] span[class="status"]'));
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('tr[class="read"] span[class="status"]'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:259]:
```suggestion
assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="read"] span[class="status"]'));
```

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 25]
```diff
@@ -241,55 +238,55 @@ assert.equal(new __helpers.CSSHelp(document).getStyle('span')?.getPropVal('displ
...
 ```js
-assert(new __helpers.CSSHelp(document).getStyle('tr[class="read"] span[class="status"]')?.backgroundImage);
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('tr[class="read"] span[class="status"]')?.backgroundImage);
 ```
 
 You should have an attribute selector to target the `span` elements with the class of `status` that are descendants of `tr` elements with the class of `in-progress`.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="in-progress"] span[class="status"]'));
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('tr[class="in-progress"] span[class="status"]'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:277]:
```suggestion
assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="in-progress"] span[class="status"]'));
```

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 26]
```diff
@@ -305,23 +302,23 @@ You should use an attribute selector to target the `span` elements with the clas
...
+assert.isNotEmpty(selector1?.height || selector2?.height);
 ```
 
 You should use an attribute selector to target the `span` elements with the class of `status` and the `span` elements with the class value starting with `rate` and set their `width` property.
 
 ```js
 const selector1 = new __helpers.CSSHelp(document).getStyle('span[class="status"], span[class^="rate"]');
 const selector2 = new __helpers.CSSHelp(document).getStyle('span[class^="rate"], span[class="status"]');
-assert(selector1?.width || selector2?.width);
+assert.exists(selector1?.width || selector2?.width);
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:313]:
```suggestion
assert.isNotEmpty(selector1?.width || selector2?.width);
```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md (hunk 15 above)
Comment (ilenia-magoni) [reply curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:313]: yep, sorry, wrong suggestion here

File: curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md [hunk 27]
```diff
@@ -211,7 +208,7 @@ assert.include(new __helpers.CSSHelp(document).getStyle('tr[class="read"]')?.bac
 You should have an attribute selector to target rows that have the class of `to-read`.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"]'));
+assert.exist(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"]'));
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md:211]:
```suggestion
assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="to-read"]'));
```

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]

**Review by majestic-owl448 — APPROVED** [review]

**Review by moT01 — APPROVED** [review]

**Pull request merged**